.pre-commit-config.yaml

```diff
@@ -7,12 +7,25 @@ repos:
       - id: end-of-file-fixer
       - id: trailing-whitespace
   - repo: https://github.com/psf/black
-    rev: 22.3.0 # This should be kept in sync with the version in requirements-dev.in
+    rev: 22.6.0 # This should be kept in sync with the version in requirements-dev.in
     hooks:
       - id: black
+        args: ["--preview"]
         language_version: python3
   - repo: https://github.com/pycqa/pydocstyle
     rev: 6.1.1 # This should be kept in sync with the version in requirements-dev.in
     hooks:
       - id: pydocstyle
+        args: ["--convention=pep257"]
         additional_dependencies: ["toml"]
+  - repo: https://github.com/pycqa/isort
+    rev: 5.10.1 # This should be kept in sync with the version in requirements-dev.in
+    hooks:
+      - id: isort
+        args: ["--profile", "black", "--filter-files"]
+        files: gnomad_constraint
+  - repo: https://github.com/PyCQA/docformatter
```

Comment:
I found some resources recommend docformatter which is able to automatically format the docstring to PEP 257. One concern I have is that when it puts summary or description that is too long to a new line, pydocstyle will raise `D205: 1 blank line required between summary line and description`, `D209: Multi-line docstring closing quotes should be on a separate line`, and `D400: First line should end with a period (not 'f')`. I only added it to the pre-commit hook for testing purpose. Please let me know your thoughts on it!

Reply:
Did `docformatter` actually work for you? I tried it on gnomad_methods when I was looking into this and it barely reformatted anything. I did 
```
docformatter --in-place --pre-summary-newline -r gnomad --wrap-descriptions 88 --wrap-summaries 0 
```
To allow the summary to be any length, but try to wrap the descriptions and it didn't wrap any of them for me. So then I tried with `--force-wrap` and it wrapped everything, but it was a complete mess

Reply:
It works for me in the pre-commit. I just changed the format of `resource_utils.py` and ran pre-commit. Let me dig into it.

Reply:
I tested the docformatter on the command line and found that `--wrap-descriptions` doesn't work. I also found that docformatter won't reformate docstrings start with `-`. It looks like docformatter won't be that helpful without `--wrap-descriptions`. Maybe we will just decide a docstring length limit and do it ourselves :(